deploy/common/central-dev-patch.yaml

```diff
@@ -0,0 +1,12 @@
+spec:
+  template:
+    spec:
+      containers:
+      - name: central
+        resources:
+          limits:
+            cpu: null
+            memory: 4Gi
+          requests:
+            cpu: 100m
```

Comment:
Same here, I feel like the reduce to 1/10th of the original resource requests might be too steep.